crates/fluvio-protocol/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "fluvio-protocol"
 edition = "2021"
-version = "0.8.3"
```

Comment:
why this need to be bump up?

Reply:
It is due to the changes in the other branch

Reply:
Yeah this is going into integration branch. We should double check when integration is moving to master that it's correct.